includes/class-wcs-att-form-handler.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * WCS_ATT_Form_Handler class
+ *
+ * @author   SomewhereWarm <info@somewherewarm.gr>
+ * @package  WooCommerce Subscribe All the Things
+ * @since    2.1.0
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Handles front-end form submissions.
+ *
+ * @class    WCS_ATT_Form_Handler
+ * @version  2.1.0
+ */
+class WCS_ATT_Form_Handler {
+
+	/**
+	 * Get the posted subscription scheme key by context.
+	 *
+	 * @param  string  $context
+	 * @param  array   $args
+	 * @return string
+	 */
+	public static function get_posted_subscription_scheme( $context, $args = array() ) {
```

Comment:
+1

Reply:
This class actually feels out of place because it doesn't really do any form handling. Once the methods are split up, it's clear they belong in existing classes.